ui/components/app/transaction-alerts/transaction-alerts.js

```diff
@@ -30,74 +34,61 @@ const TransactionAlerts = ({
         />
       )}
       {supportsEIP1559 && pendingTransactions?.length > 0 && (
-        <ActionableMessage
-          message={
-            <Typography
-              align="left"
-              className="transaction-alerts__pending-transactions"
-              margin={0}
-              tag={TypographyVariant.paragraph}
-              variant={TypographyVariant.H7}
-            >
-              <strong>
-                {pendingTransactions?.length === 1
-                  ? t('pendingTransactionSingle', [pendingTransactions?.length])
-                  : t('pendingTransactionMultiple', [
-                      pendingTransactions?.length,
-                    ])}
-              </strong>{' '}
-              {t('pendingTransactionInfo')}
-              {t('learnCancelSpeeedup', [
-                <a
-                  key="cancelSpeedUpInfo"
-                  href={ZENDESK_URLS.SPEEDUP_CANCEL}
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  {t('cancelSpeedUp')}
-                </a>,
-              ])}
-            </Typography>
-          }
-          useIcon
-          iconFillColor="var(--color-warning-default)"
-          type="warning"
-        />
+        <BannerAlert severity={SEVERITIES.WARNING}>
+          <Typography
+            align="left"
+            className="transaction-alerts__pending-transactions"
+            margin={0}
+            tag={TypographyVariant.paragraph}
+            variant={TypographyVariant.H7}
+          >
```

Comment:
@georgewrmarshall - should I remove this `Typography` also (middle alert in the following screenshot)?

![image](https://user-images.githubusercontent.com/82925669/223270749-058d2369-25f1-49a0-bf6f-4e7371fa74f6.png)

It has embedded links etc, so will need some styling fixes:

![image](https://user-images.githubusercontent.com/82925669/223271653-96324b97-8296-46f4-895d-b66350bc310d.png)

Reply:
Yes! The `Typography` has been deprecated. You could do something like this. (Note use of ButtonLink here)

`import { BannerAlert, Text, ButtonLink } from '../../component-library';`

```
<BannerAlert severity={SEVERITIES.WARNING}>
          <Text as="p">
            <strong>
              {pendingTransactions?.length === 1
                ? t('pendingTransactionSingle', [pendingTransactions?.length])
                : t('pendingTransactionMultiple', [
                    pendingTransactions?.length,
                  ])}
            </strong>{' '}
            {t('pendingTransactionInfo')}
            {t('learnCancelSpeeedup', [
              <ButtonLink
                key="cancelSpeedUpInfo"
                href={ZENDESK_URLS.SPEEDUP_CANCEL}
                rel="noopener noreferrer"
                target="_blank"
              >
                {t('cancelSpeedUp')}
              </>,
            ])}
          </Text>
        </BannerAlert>
```

Reply:
done